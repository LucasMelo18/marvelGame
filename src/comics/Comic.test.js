import renderer from 'react-test-renderer';


describe('Comic', () => {
  var subject;
  var tree;

  beforeEach(() => {
    subject = renderer.create("<Comic />");
    tree = subject.toJSON();
  });

  it('is expected to match the snapshot', () => {
    expect(tree).toMatchSnapshot();
  });
});
