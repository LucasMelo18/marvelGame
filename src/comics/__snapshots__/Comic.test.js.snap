// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comic is expected to match the snapshot 1`] = `"<Comic />"`;
